lib/active_merchant/billing/gateways/place_to_pay.rb

```diff
@@ -0,0 +1,235 @@
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    class PlaceToPayGateway < Gateway
+      self.test_url = 'https://api-co-dev.placetopay.ws/'
+      self.live_url = 'https://api-co.placetopay.ws/'
+
+      self.default_currency = 'COP'
+
+      self.supported_countries = ['COL']
+      self.supported_cardtypes = %i[visa master american_express discover]
+      self.homepage_url = 'https://www.placetopay.com/'
+      self.display_name = 'PlaceToPay'
+
+      STANDARD_ERROR_CODE_MAPPING = {}
+
+      ISO_FORMAT_DATE = '%Y-%m-%dT%H:%M:%S%:z'
+
+      def initialize(options = {})      
+        requires!(options, :login, :secret_key)
+        @login, @secret_key = options.values_at(:login, :secret_key)
+        super
+      end
+
+      def purchase(money, payment, options = {})
+        post = {}
+        add_auth(post, options)
+        add_payer(post, options)
+        add_payment(post, options)
+        add_instrument(post, payment, options)
+        commit(:post, 'gateway/process', post)
+      end
+
+      def capture(money, authorization, options = {})
+        post = {}
+        add_auth(post, options)
+        add_amount(post, options)
+        post[:reference] = options[:reference]
+        commit(:post, 'gateway/search', post)
+      end
+
+      def void(authorization, options = {})
+        post = {}
+        add_auth(post, options)
+        post[:internalReference] = options[:internalReference]
+        commit(:post, 'gateway/query', post)
+      end
+
+      def refund(money, authorization, options = {})
+        post = {}
+        add_auth(post, options)
+        post[:internalReference] = options[:internalReference]
+        post[:authorization] = authorization                
+        post[:action] = 'refund' #Allowed values: reverse refund process void dispersion pre_authorization checkin checkout reauthorization 
+        commit('gateway/transaction', post)
+      end
+
+
+      def supports_scrubbing?
+        true
+      end
+
+      def scrub(transcript)
+        transcript.
+          gsub(%r((tranKey)\w+), '\1[FILTERED]').
+          gsub(/(number\\?":\\?")(\d*)/, '\1[FILTERED]').
+          gsub(/(cvv\\?":\\?")(\d*)/, '\1[FILTERED]')
+      end
+
+      private
+
+      def add_customer_data(post, options); end
+
+      def add_address(post, creditcard, options); end
+
+      def add_invoice(post, money, options)
+        post[:amount] = amount(money)
+        post[:currency] = (options[:currency] || currency(money))
+      end
+
+      def parse(body)
+        {}
+      end
+
+      def build_url(action, base)
+        url = base
+        url += action
+        url
+      end
+
+      def commit(method, action, parameters)
+        base_url = (test? ? test_url : live_url)
+        url = build_url(action, base_url)
+
+        # puts "URL: " + url
```

Comment:
puts statements and other commented out lines should be removed if not needed

Reply:
Done!